src/sentry/plugins/base/v1.py

```diff
@@ -320,8 +318,9 @@ def get_view_response(self, request, group):
         if not isinstance(response, Response):
             raise NotImplementedError("Use self.render() when returning responses.")
 
-        event = group.get_latest_event() or Event()
-        event.group = group
+        event = group.get_latest_event()
```

Comment:
The event instance is returned in the response. So now it can be None while in the old implementation there was always an object returned. Is this change intentional ?